src/project/components/addProject/addProject.reducers.js

```diff
@@ -0,0 +1,15 @@
+import * as constants from './addProject.actionTypes.js'
+
+export default function (state = { addProjectDialog: {
+    open: false,
+} }, action) {
+    switch (action.type) {
+    case constants.PROJECT_CREATED:
+    case constants.ADD_PROJECT_DIALOG_TOGGLE:
+        return Object.assign({}, state, {
+            addProjectDialog: Object.assign({}, state.addProjectDialog, { open: action.open }),
```

Comment:
I saw the object spread in the babel config file. 
Therefore we can change:
```javascript
Object.assign({}, state.addProjectDialog, {open:action.open })
```

to:
```javascript
{...state.addProjectDialog, {open: action.open}}

```